README.md

```diff
@@ -3,67 +3,42 @@
 ## Introduction
 This constitutes the official documentation of the "Pay with Toonie" online payments solution.
 
-In its current implementation the integration is composed of two different implementations, both required:
-- **REST API**: Authenticated Serverside integration to initiate the Payment Session to be passed to the client-side application
-- **Vanilla JS SDK**: Client-side integration of the "Pay with Toonie Button" User Experience, able to handle QR generation, polling and communication with the payment API.
+You can see and try a full implementation example using the file in [full_example](samples/full_example) folder
 
-![Payment Buttons](imgs/buttons-page.png)
-
-![Pay With Toonie](imgs/sample_qr.jpg)  ![Pay With Toonie](imgs/sample_qr_success.jpg) 
+<img src="imgs/buttons_page.png" alt="Image description" width="508" height="649">
+<img src="imgs/sample_qr.jpg" alt="Image description" width="284" height="386">
+<img src="imgs/sample_qr_success.jpg" alt="Image description" width="332" height="386">
 
 ## REST API Integration
 
-### Authentication
-
-In order to initialize a new payment session, it is necessary to perform a very basic integration to an Authentication endpoint.
-
-```js
-// Auth to get token
-const tokenRes = await fetch("https://<ENVIRONMENT_AUTH_URL>/auth/realms/toonie/protocol/openid-connect/token", {
-    method: "POST",
-    headers: {
-        "content-type": "application/x-www-form-urlencoded",
-    },
-    body: new URLSearchParams({
-        "grant_type": "password",
-        "client_id": "pay-with-toonie",
-        "username": "customerusername",
-        "password": "customerpassword",
-    })
-});
-```
-
-In this first version of the integration, the only supported authentication method is `username/password`.
-
->*Note: Authentication via a combination of `APIKey/APISecret` is currently being developed.*
+In its current implementation the integration requires an Authenticated Serverside integration to initiate the Payment Session to be passed to the client-side application.
 
 ### Payment Session Initialization
 
 To complete the initialization of a new payment session you need to call the endpoint to create it, passing some parameters like an amount, a currency and a reason.
-You also need to pass it a success and an error url parameters.
+
+You also need to pass it a success and an error url parameters where the user will be sent after the payment.
+
 You can use the `{PAYMENT_SESSION_ID}` placeholder anywhere in your url, it will be replaced in our system with the right value.
 
 ```js
 // Create a payment session
 const createPaymentSession = async (amount, currency, reason) => {
-  const tokenData = await getTokenData();
-
   const res = await fetch("https://<ENVIRONMENT_API_URL>/acquiring/v1/payment", {
-    method: "POST",
+    method: 'POST',
```

Comment:
if we have doublequotes everywhere, why do we have single here?

Reply:
updated all the codes, replacing single quotes with double quotes